README.md

```diff
@@ -60,7 +61,7 @@ module "eks" {
   version = "~> 18.0"
 
   cluster_name    = "my-cluster"
-  cluster_version = "1.22"
+  cluster_version = "1.24"
```

Comment:
in the future, can we do these sort of changes in a different PR?
a card that mentions karpenter, and all the sudden there is a major change of the cluster version, is not what anyone would expect

Reply:
I think this is just to the readme so probably not a big deal

Reply:
The PR description contains all the necessary details, it just seems some have not taken the time to read it 😉

Reply:
I'm sorry. I seem to not have notifications for PR, only releases and missed this PR when it was initially opened. And when I saw the release email, I clicked in the commit id rather than open the PR so never got to read the description.

Reply:
all good! I do try to put the relevant bits in the PR body with associated links as best as I can. its not always perfect though